gtsfm/configs/unified.yaml

```diff
@@ -69,11 +69,13 @@ SceneOptimizer:
 
     rot_avg_module:
       _target_: gtsfm.averaging.rotation.shonan.ShonanRotationAveraging
+      weight_by_inliers: True
 
     trans_avg_module:
       _target_: gtsfm.averaging.translation.averaging_1dsfm.TranslationAveraging1DSFM
       robust_measurement_noise: True
       projection_sampling_method: SAMPLE_INPUT_MEASUREMENTS
+      reject_outliers: True
```

Comment:
Unintended change.

Reply:
This was already the default in the corresponding constructor args, but makes it more explicit now.